### PR TITLE
Pull in some packaging changes from debian

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: contrib/debian/rules [hunk 1]
```diff
@@ -70,6 +70,11 @@ override_dh_install:
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_test_ble.so
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_invalid.so
 	rm -f debian/fwupd/etc/fwupd/remotes.d/fwupd-tests.conf
+	mv debian/fwupd/usr/share/man/man1 debian/fwupd/usr/share/man/man8
+	for file in debian/fwupd/usr/share/man/man8/*.1; do \
+		file8=$${file%%.1}.8; \
+		mv -v $$file $$file8; \
```
Comment (hughsie) [inline contrib/debian/rules:76]: is this something we should do upstream?

Comment (superm1) [inline contrib/debian/rules:76]: the logic behind this is https://lintian.debian.org/tags/manual-page-for-system-command?version=2.104.244

Comment (superm1) [reply contrib/debian/rules:76]: I was wondering the same thing.  At least wanted to sync up the packaging for now, and can revert it if we change it upstream too.

Comment (hughsie) [reply contrib/debian/rules:76]: Hmm, not convinced e.g. `fwupdmgr` is designed to be run by end users, rather than sysadmins -- you don't even need root.

Comment (superm1) [reply contrib/debian/rules:76]: I guess that's true, I'm not sure why this lintian warning came up.

**Review by hughsie — DISMISSED** [review]
Maybe squash down?

**Comment by superm1** [comment]
> Maybe squash down?

I'll add trivial tags to them, but I want to keep them as individual commits to match where they came from in Debian for future merges.

**Review by hughsie — APPROVED** [review]

**Pull request merged**